source/hardware/connecting-to-the-serial-console.rst

```diff
@@ -93,6 +93,13 @@ port, start PuTTY and then set the following parameters:
 The session may be saved for later use. Click **Open** to start the
 serial console session.
 
+MINICOM
+~~~~~~~
+
+..code-block:: console
```

Comment:
One more little change, it's missing a space after ``..``, so it needs to be ``.. code-block:: console``. Otherwise it looks good.